src/tests/ftest/cat_recovery/ddb.py

```diff
@@ -0,0 +1,666 @@
+
+                    # Prepare the data to insert into the akey.
+                    data_str = " ".join(
+                        [SAMPLE_DATA, str(obj_index), str(dkey_index),
+                         str(akey_index)]).encode("utf-8")
+                    self.data_list.append(
+                        create_string_buffer(value=data_str, size=len(data_str)))
+                    c_size = ctypes.c_size_t(ctypes.sizeof(self.data_list[-1]))
+
+                    # Insert dkeys, akeys, and the data.
+                    self.ioreqs[-1].single_insert(
+                        dkey=self.dkeys[-1], akey=self.akeys[-1],
+                        value=self.data_list[-1], size=c_size)
+
+    def get_vos_file_path(self):
+        """Get the VOS file path.
+
+        Returns:
+            str: VOS file path such as /mnt/daos/<pool_uuid>/vos-0
+
+        """
+        hosts = NodeSet(self.hostlist_servers[0])
+        vos_path = os.path.join("/mnt/daos", self.pool.uuid.lower())
+        command = " ".join(["sudo", "ls", vos_path])
+        cmd_out = run_pcmd(hosts=hosts, command=command)
+
+        vos_file = None
+        for file in cmd_out[0]["stdout"]:
+            # Assume the VOS file has "vos" in the file name.
+            if "vos" in file:
+                vos_file = file
+                break
+
+        if not vos_file:
+            self.fail(
+                "vos file wasn't found in /mnt/daos/{}".format(self.pool.uuid.lower()))
+        else:
+            self.log.info("vos_file: %s", vos_file)
+
+        return vos_file
+
+    def copy_remote_to_local(self, remote_file_path):
+        """Copy the given file from the server node to the local test node and retrieve
+        the original name.
+
+        Args:
+            remote_file_path (str): File path to copy to local.
+        """
+        # Use clush --rcopy to copy the file from the remote server node to the local test
+        # node. clush will append .<server_hostname> to the file when copying.
+        args = "--rcopy {} --dest {}".format(remote_file_path, self.test_dir)
+        clush_command = get_clush_command(hosts=self.hostlist_servers[0], args=args)
+        try:
+            run_command(command=clush_command)
+        except DaosTestError as error:
+            raise CommandFailure(
+                "ERROR: Copying {} from {}: {}".format(
+                    remote_file_path, self.hostlist_servers[0], error)) from error
+
+        # Remove the appended .<server_hostname> from the copied file.
+        current_file_path = "".join([remote_file_path, ".", self.hostlist_servers[0]])
+        mv_command = "mv {} {}".format(current_file_path, remote_file_path)
+        try:
+            run_command(command=mv_command)
+        except DaosTestError as error:
+            raise CommandFailure(
+                "ERROR: Moving {} to {}: {}".format(
+                    current_file_path, remote_file_path, error)) from error
+
+    def test_ls(self):
```

Comment:
Recommend making the test name more specific / less prone to existing elsewhere.
Similar for other test methods in this file
```suggestion
    def test_recovery_ddb_ls(self):
```

Reply:
Done